client/src/App.js

```diff
@@ -23,10 +44,11 @@ export default class App extends Component {
     return (
         <div>
           <SavedList list={this.state.savedList} />
-          {/* <Route path='/' exact component={MovieList} /> */}
-          <Route path='/movies/:id' component={MovieCard} />
+          <Route path='/' exact render={ (props) => <MovieList {...props} movies={this.state.movies} /> } />
+          <Route path='/movies/:id' render={ (props) => <MovieCard {...props} movies={this.state.movies} /> } />
         </div>
     );
   }
 }
 
+export default App;
```

Comment:
If you wanted to use withRouter, this line would be `export default withRouter(App)`